prerendering.bs

```diff
@@ -658,6 +673,33 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+[=Accept-CH cache=], which is owned by user agent as a global storage, should not be modified by [=prerendering browsing context=].
+
+Instead, each [=prerendering browsing context=] maintains its <dfn>prerender scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields. 
+
+Similar to <a spec=CLIENT-HINTS-INFRASTRUCTURE> add a new Accept-CH cache entry</a>,
+to <dfn>add a new prerender Accept-CH cache entry</dfn> to the [=prerender scoped Accept-CH cache=],
+given an |origin| and a [=client hints set=] |hintSet|,
+[=map/set=] [=prerender scoped Accept-CH cache=][|origin|] to |hintSet|.
```

Comment:
Even though it's trivial this might be clearer to do in a `<div algorithm>`. I'm also confused as to why an algorithm var is linked to another dfn.

Reply:
Thanks! I moved it into a <div algorithm>.

Do you mean that I should not use `Similar to <a spec=CLIENT-HINTS-INFRASTRUCTURE> add a new Accept-CH cache entry</a>,`?
Sorry, I haven't figured out whether I should use <a spec=""> or not.. Could you help point me to the instructions?